backend/handler/handler.js

```diff
@@ -0,0 +1,75 @@
+const dotenv = require('dotenv');
+const fs = require('fs');
+const Host = require('./Classes/Host');
+const Room = require('./Classes/Room');
+
+const dataLoc = 'handler/data.json';
```

Comment:
Is there an alternative to reading the current room state out of a file? 
Otherwise, I would prefer it if there was a JSON file for each room. These could be called `'handler/ABCDEF.json' for example (assuming ABCDEF was the name of the room). 
If this causes issues we could check for free database alternatives such as

- [FaunaDB](https://fauna.com/) (they offer a generous free tier), 
- [Heroku Postgres or Redis](https://www.heroku.com/) (they have a free tier too)
- any other service that we might stumple upon whilst investigating

Reply:
All of those are a option, better to discuss in a call

Reply:
@Michiocre should we add an issue to explore a database option?

Reply:
Sure

Reply:
referenced in #236